### PR TITLE
Add support for minikube drivers

### PR DESCRIPTION
**Pull request by alacuku**
This PR adds the necessary bits to generate the `config` files for minikube. When running `scrape_and_generate` script with the appropriate input it will generate config files containing something similar to:
```yaml
kernelversion: 1_1.26.0
kernelrelease: 5.10.57
target: minikube
architecture: amd64
output:
  module: output/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/falco_minikube_5.10.57_1_1.26.0.ko
  probe: output/b7eb0dd65226a8dc254d228c8d950d07bf3521d2/falco_minikube_5.10.57_1_1.26.0.o
kernelconfigdata: Q09ORklHX0ZBTk9USUZZPXkKQ09ORklHX0tQUk9CRV9FVkVO...
```
The `kernel-crawler` has already been updated falcosecurity/kernel-crawler#31 as well as `driverkit` falcosecurity/driverkit#189. 

Furthermore this PR tries to simplify how `kernel_version` and `kernel_release` are handled in `generate` script. The idea is to pass both kernel version and release as separate arguments. The current implementation uses a variable called `TARGET_KERNEL` which contains both the release and version (TARGET_KERNEL="{release}_{version}. Then `TARGET_KERNEL` is again split in two different variables containing the release and version. In some case it becomes cumbersome to split the `TARGET_KERNEL` when multiple `_` are present in the string. For this reason having them separated in `generate` script is preferred and we just join the strings when needed.

### CODE REVIEW

**Comment by maxgio92** [comment]
Hey @alacuku thank you. Looks good to me.
In the meantime I'd ask you to wait the previous #766 is addressing part of this goal so this PR likely will need to be rebased on that.

/hold

File: driverkit/utils/generate [hunk 1]
```diff
@@ -16,14 +16,11 @@ declare -A output=()
 
 CURRENT_DIR="$(pwd)"
 
-while getopts ":a:k:d:v:h:" arg; do
+while getopts ":a:k:d:v:h:c:r:t:" arg; do
```
Comment (FedeDP) [inline driverkit/utils/generate:19]: I'd leave `-k` as `TARGET_KERNEL` so that we have a smaller diff :)

**Review by FedeDP — COMMENTED** [review]
Aside from 1 minor comment, LGTM!

**Review by FedeDP — APPROVED** [review]
My 2c :)

**Comment by maxgio92** [comment]
Thank you @alacuku!
/lgtm

**Review by maxgio92 — APPROVED** [review]
/approve

This brings the missing pieces too in DBG config generation @alacuku, thanks.

**Pull request merged**